_config.yml

```diff
@@ -3,7 +3,7 @@
 
 title: Bayesian Modeling and Computation in Python
 author: Martin, Kumar, Lao
-exclude_patterns: [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints", "**/README.md"]
+exclude_patterns: [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints", "**/README.md", "README.md", "Errata.md", "test"]
```

Comment:
Whats test?

Reply:
No idea, haha. I assumed it was there for some reason. Probably I added at some point to test something. I already remove it.